llvm/include/llvm/Object/ELF.h

```diff
@@ -207,6 +209,43 @@ bool isSectionInSegment(const typename ELFT::Phdr &Phdr,
          checkSectionVMA<ELFT>(Phdr, Sec);
 }
 
+template <bool Is64>
```

Comment:
Thanks for lifting this out. Possibly worth a comment describing HdrHandler and EntryHandler. For example:
```
// The HdrHandler is called once with the number of relocations and whether the relocations have addends.
// The EntryHandler is called once per decoded relocation.

Reply:
thx for the suggestion. adopted